lib/matplotlib/backend_bases.py

```diff
@@ -2935,6 +2935,17 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
     def remove_rubberband(self):
         """Remove the rubberband."""
 
+    def draw_whiskers(self, event, x0, y0, x1, y1, ws):
+        """
+        Draw line with whiskers to indicate single axis zoom
+
+        We expect that ``x0 == x1`` or ``y0 == y1``. Else nothing will draw
+        `ws` is the whisker size in pixels.
```

Comment:
```suggestion
        *ws* is the whisker size in pixels.
```
We mark parameters this way. Single ticks will try to link somewhere and induce a failure in the doc build.